### PR TITLE
Move reward off contribution details

### PR DESCRIPTION
**Pull request by diogob**

### CODE REVIEW

File: spec/lib/mocks/reward-details.mock.js [hunk 1]
```diff
@@ -18,6 +18,6 @@ beforeAll(function() {
   };
 
   jasmine.Ajax.stubRequest(new RegExp("("+apiPrefix + '\/reward_details)'+'(.*)')).andReturn({
-    'responseText' : JSON.stringify(ProjectPostDetailsMockery())
+    'responseText' : JSON.stringify(RewardDetailsMockery())
```
Comment (vicnicius) [inline spec/lib/mocks/reward-details.mock.js]: hehehe

Comment (diogob) [inline spec/lib/mocks/reward-details.mock.js]: no, this was correct before we started.

File: src/c/admin-reward.js [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+
+      return {
+        loader: l,
+        reward: reward
+      };
+    },
+    view: (ctrl, args) => {
+      const reward_id = args.contribution.reward_id,
+            reward = ctrl.reward() || {},
+            available = parseInt(reward.paid_count) + parseInt(reward.waiting_payment_count);
```
Comment (diogob) [inline src/c/admin-reward.js]: the problem was here when reward was undefined!
So I added the ```|| {}``` above ;)
